package.json

```diff
@@ -59,7 +64,8 @@
   "husky": {
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-commit": "lint-staged"
+      "pre-commit": "lint-staged",
+      "pre-push": "yarn test"
```

Comment:
I prefer to not have this hook because it slows down the development process. Not a general rule, but for frontend tests, which tend to be slower than backend tests, we can do without

Reply:
Yeah, fair enough. I'm dropping it.